fronts-client/src/components/card/Card.tsx

```diff
@@ -258,7 +258,8 @@ class Card extends React.Component<CardContainerProps> {
                 {...getNodeProps()}
                 onDelete={this.onDelete}
                 onAddToClipboard={this.handleAddToClipboard}
-                /* No need for an OnClick here - there are no editable forms */
+                // Chef has overrides so we need to edit it
+                onClick={isUneditable ? undefined : () => onSelect(uuid)}
```

Comment:
Oh this was removed to handle `!` issue on overview but we do need it for `chef`, glad sorted as it was missed checking by us.